### ISSUE TITLE
[dipolefit] no sphinx documentation (Trac #1445)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1437]
Good documentation is now deemed essential.

Migrated from https://code.icecube.wisc.edu/ticket/1445
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "Good documentation is now deemed essential.", 
    "reporter": "david.schultz", 
    "cc": "olivas", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[dipolefit] no sphinx documentation", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-11-24T23:14:58", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160119T19:17:35: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20160216T10:42:41: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "assigned" to "closed"
* kjmeagher commented:

> In changeset 141912:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141912"
> migrate documentation to sphinx
> fixes #1437
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
